src/main/java/org/apache/sling/feature/cpconverter/ContentPackage2FeatureModelConverter.java

```diff
@@ -328,7 +328,6 @@ private void secondPass(@NotNull Collection<VaultPackage> orderedContentPackages
                     // finally serialize the Feature Model(s) file(s)
 
                     aclManager.addRepoinitExtension(assemblers, featuresManager);
-                    bundleSlingInitialContentExtractor.addRepoInitExtension(assemblers, featuresManager);
```

Comment:
hm..... correct me if i am wrong.... looking at the whole class it seems to me that you are removing the only place where the `bundleSlingInitialContentExtractor` is actually being invoked making changes to the sling initial content.
am i missing something?

or phrased differently:
what is left after removing that statement is
- line 293: bundleSlingInitialContentExtractor.reset()
- line 342: bundleSlingInitialContentExtractor.reset()
- line 528: setBundleSlingInitialContentExtractor(BundleSlingInitialContentExtractor) invoked from the launcher.

and what is the reset doing now? and why do you still need it here?

it is effectively used in `DefaultEntryHandlersManager` which passes it to `BundleEntryHandler`, no? why still referencing it in the converter?

it might all be ok and needed but in that case i believe a bit of comment explaining why you are resetting it, might help. alternatively, leave that all to the handlersmanager.